pages/management/ManagementHandler.php

```diff
@@ -144,11 +144,13 @@ public function context($args, $request)
             ],
         ]);
 
+        $showUpgradeWarning = Config::getVar('general', 'show_upgrade_warning');
+
         // Interact with the beacon (if enabled) and determine if a new version exists
-        $latestVersion = VersionCheck::checkIfNewVersionExists();
+        if ($showUpgradeWarning) $latestVersion = VersionCheck::checkIfNewVersionExists();
```

Comment:
This line should also be always executed, for now the [Beacon](https://pkp.sfu.ca/software/ojs/usage-data/) depends on this, therefore disabling the upgrade warning shouldn't disable the Beacon.